tests/factories.py

```diff
@@ -100,9 +109,8 @@ class Meta:
         model = models.Theme
 
     # TODO - may need to be changed once ML pipeline is in
-    label = faker.sentence()
+    label = generate_dummy_topic_label()
```

Comment:
This needs to be a `LazyAttribute` or it will always return the same thing.

```suggestion
    label = factory.LazyAttribute(lambda _o: generate_dummy_topic_label())
```

As now:
<img width="316" alt="Screenshot 2024-04-14 at 08 23 30" src="https://github.com/i-dot-ai/consultation-analyser/assets/642279/bd557ed3-1be4-4df8-be0e-2d1e16c5d23e">

As LazyAttribute:

<img width="323" alt="Screenshot 2024-04-14 at 08 24 28" src="https://github.com/i-dot-ai/consultation-analyser/assets/642279/d3b85d18-4701-4e7c-9f2d-5fd8bfe5255f">

Reply:
Now updated.